tests/conftest.py

```
@@ -391,7 +451,13 @@ def remote_manifest():
 def remote_bytes(relative_path):
     """Raw bytes of a file on the remote, as stored (encrypted)."""
     result = subprocess.run(
-        ["docker", "exec", REMOTE_CONTAINER, "cat", f"{REMOTE_BACKUP_DIR}/{relative_path}"],
+        [
+            "docker",
+            "exec",
+            REMOTE_CONTAINER,
+            "cat",
+            f"{REMOTE_BACKUP_DIR}/{relative_path}",
+        ],
```

Ruff (0.16.1)

[error] Starting a process with a partial executable path

(S607)